appveyor.yml

```diff
@@ -0,0 +1,22 @@
+os: Visual Studio 2017
+
+build_script:
+- call "%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Community\\VC\\Auxiliary\\Build\\vcvarsall.bat" x64
+- IF defined APPVEYOR_REPO_TAG_NAME ( SET VersionDefinition=/DSW_VER=L\"%APPVEYOR_REPO_TAG_NAME%\" )
+- mkdir out
+- cl /nologo /O2 /W4 /WX /Ob2 /Oi /Oy /Gs- /GF /Gy /Tc main.c /Fe:out\Launcher.exe Advapi32.lib Shell32.lib %VersionDefinition%
+- FOR /D /r %%D in ("res/*") DO (
+    rc /nologo res\%%~nxD\res.rc &&
+    cl /nologo /O2 /W4 /WX /Ob2 /Oi /Oy /Gs- /GF /Gy /Tc main.c /Fe:out\%%~nxD.exe Advapi32.lib Shell32.lib res\%%~nxD\res.res %VersionDefinition%)
+- ps: Compress-Archive -DestinationPath out\icons.zip -Path  (Get-ChildItem -Path out -Exclude Launcher.exe)
+
+artifacts:
+  - path: 'out\icons.zip'
+  - path: 'out\Launcher.exe'
+
+deploy:
+ - provider: GitHub
+   auth_token:
+     secure: FAHMAdf6tEGceEb9J7odXO2KYZb8eFnD+dHOX9mE7FRJhw9GEO1wMoynJ5wl38to
```

Comment:
To obtain suitable token:

*  Create a 'personal access token' with just one scope checkbox checked: `public_repo` https://github.com/settings/tokens <br/><img src="https://user-images.githubusercontent.com/3840695/52527184-0d072280-2ccd-11e9-9520-848a691efb9f.png" alt="screenshot-1" width="42%" />
* Go to https://ci.appveyor.com/tools/encrypt, login with GitHub account and encrypt.

To add project, visit https://ci.appveyor.com/account/yuk7/projects/new and select wsldl.